SpatialGDK/Source/SpatialGDKServices/Private/SpatialGDKServicesModule.cpp

```diff
@@ -23,9 +23,23 @@ FLocalDeploymentManager* FSpatialGDKServicesModule::GetLocalDeploymentManager()
 	return &LocalDeploymentManager;
 }
 
-FString FSpatialGDKServicesModule::GetSpatialOSDirectory()
+FString FSpatialGDKServicesModule::GetSpatialOSDirectory(FString RelativePath)
```

Comment:
```suggestion
FString FSpatialGDKServicesModule::GetSpatialOSDirectory(const FString& RelativePath)
```

Reply:
I'm not sure about RelativePath either. What about AppendPath? RelativePath feels like it would be prepended to it.

Reply:
Done